src/WebJobs.Extensions.CosmosDB/WebJobs.Extensions.CosmosDB.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.20.1" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.27" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.32-dev637683692850667066" />
```

Comment:
The referencing of these dev packages is temporary just to get a PR out right? We can't check in until we have actual published packages.

Reply:
correct - temporary stop-gap to get the pipelines moving.